### PR TITLE
2.3- Add Backup and restore tech note

### PR DESCRIPTION
**Pull request by zkayyali812**

### CODE REVIEW

File: stories/assembly-aws-tech-notes.adoc [hunk 1]
```diff
@@ -0,0 +1,11 @@
+ifdef::context[:parent-context: {context}]
+
+[id="assembly-aws-tech-notes"]
+= Tech Notes
```
Comment (ianf77) [inline stories/assembly-aws-tech-notes.adoc:4]: Technical

File: stories/assembly-aws-tech-notes.adoc [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+:context: aws-tech-notes
+
+{AAPonAWS} is a self-managed deployment. Below are tech notes regarding the {AAPonAWS} deployment.
```
Comment (ianf77) [inline stories/assembly-aws-tech-notes.adoc:8]: technical

File: stories/topics/tech-note-aws-backupandrestore.adoc [hunk 3]
```diff
@@ -0,0 +1,12 @@
+[id="tech-note-aws-backupandrestore"]
+
+= Tech Notes - Backup and Restore
```
Comment (ianf77) [inline stories/topics/tech-note-aws-backupandrestore.adoc:3]: Technical

File: stories/topics/tech-note-aws-backupandrestore.adoc [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+
+{AAPonAWS} can be backed up and restored via the operational container. To ensure a successful restore, ensure that the following secrets in secret manager do not have the special character '\' contained within it. Updating the secret value of the noted secrets to remove the '\' character is necessary to ensure a successful restore.
```
Comment (ianf77) [inline stories/topics/tech-note-aws-backupandrestore.adoc:5]: via -> using

**Review by ianf77 — COMMENTED** [review]
A few comments. I wonder why a Technical note is necessary and the information couldn't just be included in the Backup and Restore assembly?

**Comment by zkayyali812** [comment]
Hey @ianf77 ,

The reason they are not directly in the backup and restore section, is we may possibly update our tech notes section further later with other notes, which may not be related to backup and restore, as we discover them during our release.

File: stories/topics/tech-note-aws-backupandrestore.adoc (hunk 4 above)
Comment (zkayyali812) [reply stories/topics/tech-note-aws-backupandrestore.adoc]: updated

File: stories/assembly-aws-tech-notes.adoc (hunk 1 above)
Comment (zkayyali812) [reply stories/assembly-aws-tech-notes.adoc]: updated

**Review by ianf77 — APPROVED** [review]
LGTM

**Comment by ianf77** [comment]
Shall I merge?

**Comment by zkayyali812** [comment]
@ianf77 Yes! When you are ready, this should be merged and published

**Pull request merged**